testnets/bitcannadev2/assetlist.json

```diff
@@ -0,0 +1,28 @@
+{
+    "$schema": "../../assetlist.schema.json",
+    "chain_name": "bitcanna-dev-6",
```

Comment:
chain_name != chain_id. Please change chain_name to 'bitcannadev2', as it must match the folder name exactly. Same goes for the chain_name property of the chain.json file as well.